SU-SWS/stanford_profile
# D8CORE-943 Added more image styles for wysiwyg

# READY FOR REVIEW

# Summary
- Added more image styles to be allowed in the wysiwyg.

# Need Review By (Date)
- 3/13

# Urgency
- low

# Steps to Test
1. checkout this branch
1. drush cim
1. in a wysiwyg embed an image
1. choose from the various styles
1. verify they do what you'd think they should do.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
